README.md

```diff
@@ -1,21 +1,61 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
-## Use the client-side tool
+## scalar-admin client tool
+
+The scalar-admin client tool is used to manage the admin interface implemented applications.
+It will help to `PAUSE` and `UNPAUSE` the application without losing the transaction.
+
+### Installation
```

Comment:
The current doc looks like we need to build to use it, which is not true.
Please restructure the whole doc.
(I think the building part should come later, but please think)

Reply:
I will recreate it.

Reply:
Document updated like scalar-schema tool doc.
As in the scalar-schema tool, the building part is placed after the installation step.